### PR TITLE
updatable capsule

### PR DESCRIPTION
**Pull request by avin**
added missed `updatable` param

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# editorconfig.org
+root = true
```
Comment (deltakosh) [inline .editorconfig:2]: why this change? I guess this file should be removed

**Review by deltakosh — APPROVED** [review]

File: .editorconfig (hunk 1 above)
Comment (avin) [reply .editorconfig:2]: it defines setups for editors (analogue of .vscode/settings.json, but without reference to a specific one)

Comment (deltakosh) [reply .editorconfig:2]: ok why not:)

you have some checks failing and I can merge
